pkg/descheduler/strategies/duplicates.go

```diff
@@ -44,6 +44,17 @@ func RemoveDuplicatePods(
 	nodes []*v1.Node,
 	podEvictor *evictions.PodEvictor,
 ) {
+	var priorityClass string
+	if strategy.Params != nil {
+		priorityClass = strategy.Params.ThresholdPriorityClassName
+	}
+	priority, err := podEvictor.SetPriorityThresholdFromPriorityClass(ctx, client, priorityClass)
+	if err != nil {
+		klog.V(1).Infof("Error setting priority threshold, %#v", err)
+		return
+	}
+	klog.V(1).Infof("Running RemoveDuplicatePods with priority threshold %d", priority)
```

Comment:
I think we should only log the threshold if it's being set to something that's not default.